website/server/libs/spells.js

```diff
@@ -163,14 +163,32 @@ async function castSpell (req, res, { isV3 = false }) {
       task: results[1],
     });
   } else if (targetType === 'self') {
-    await castSelfSpell(req, user, spell, quantity);
-
-    let userToJson = user;
-    if (isV3) userToJson = await userToJson.toJSONWithInbox();
-
-    res.respond(200, {
-      user: userToJson,
-    });
+    // Check if chilling frost or stealh skill has been previously casted or not.
+    // See #12361 for more details.
+    const spellName = spell.key;
+    const incompleteDailiesDue = await Tasks.Task.find({
```

Comment:
instead of loading the tasks you can simply count the number of results by doing `.countDocuments().exec()` instead of `.find()`

Reply:
Modified. This was a better idea. Thanks!